core/storage/base_model/gae_models.py

```diff
@@ -236,41 +229,75 @@ def get_multi(cls, entity_ids, include_deleted=False):
                     entities[i] = None
         return entities
 
-    def put(self, update_last_updated_time=True):
-        """Stores the given ndb.Model instance to the datastore.
+    def update_timestamps(self, update_last_updated_time):
```

Comment:
Should this method be private?

(If so make sure to update the docstrings above too -- the ones that refer to it.)

Reply:
Yep, done.